Homework-01/Task-05/src/test/java/by/bsu/tat/task05/TriangleTest.java

```
@@ -0,0 +1,346 @@
+                {Double.POSITIVE_INFINITY, Double.NEGATIVE_INFINITY, 0},
+                {Double.POSITIVE_INFINITY, Double.NEGATIVE_INFINITY, 1},
+                {Double.POSITIVE_INFINITY, Double.NEGATIVE_INFINITY, Double.POSITIVE_INFINITY},
+
+                {Double.POSITIVE_INFINITY, -1, Double.NaN},
+                {Double.POSITIVE_INFINITY, -1, Double.NEGATIVE_INFINITY},
+                {Double.POSITIVE_INFINITY, -1, -1},
+                {Double.POSITIVE_INFINITY, -1, 0},
+                {Double.POSITIVE_INFINITY, -1, 1},
+                {Double.POSITIVE_INFINITY, -1, Double.POSITIVE_INFINITY},
+
+                {Double.POSITIVE_INFINITY, 0, Double.NaN},
+                {Double.POSITIVE_INFINITY, 0, Double.NEGATIVE_INFINITY},
+                {Double.POSITIVE_INFINITY, 0, -1},
+                {Double.POSITIVE_INFINITY, 0, 0},
+                {Double.POSITIVE_INFINITY, 0, 1},
+                {Double.POSITIVE_INFINITY, 0, Double.POSITIVE_INFINITY},
+
+                {Double.POSITIVE_INFINITY, 1, Double.NaN},
+                {Double.POSITIVE_INFINITY, 1, Double.NEGATIVE_INFINITY},
+                {Double.POSITIVE_INFINITY, 1, -1},
+                {Double.POSITIVE_INFINITY, 1, 0},
+                {Double.POSITIVE_INFINITY, 1, 1},
+                {Double.POSITIVE_INFINITY, 1, Double.POSITIVE_INFINITY},
+
+                {Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, Double.NaN},
+                {Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, Double.NEGATIVE_INFINITY},
+                {Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, -1},
+                {Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, 0},
+                {Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, 1},
+                {Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY},
+
+                /**/
+                {Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE},
```

This is positive. Moving it to negative hides serious error in the code. It's not accepted to hide error by moving positive test case to negative - code had to be fixed.